shopware/paas-meta/6.4/.platform/services.yaml

```diff
@@ -14,3 +14,8 @@ rabbitmq:
 #opensearch:
 #    type: opensearch:1.2
 #    disk: 256
+
+# Change the disk space according to your needs for a.o. media files
+fileshare:
+    type: network-storage:2.0
+    disk: 4096
```

Comment:
As we move some files to this fileshare. What happens with the files that already exist? (the config before)

Reply:
The new mounted share will be empty. When updating, this will have to be migrated manually. I don't see a good option which would allow for an automatic migration for existing projects.

I think the recipe is only ran when a new project is created?

Reply:
also when you run `composer recipe:update`

Reply:
Hmm that's bad news. I think the mounts surely _have_ to be updated to use service mounts so the data is shared between the different services and workers.

I have asked PSH about this, but unfortunately, there is no automated way at the moment to migrate this data.

How would you propose that we go further here? Is it possible to change the version or sth so that new projects would use the new template, but current users won't update it automatically/without knowing the consequences?

https://docs.platform.sh/add-services/network-storage.html#how-can-i-migrate-a-local-storage-to-a-network-storage